src/components/atomic/dynamic-form/VCheckbox.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+    <div>
+        <v-form-label v-bind="labelProps">
```

Comment:
I'm fine to go ahead with this at the moment, but I'm still not sure about it as it means you have to map labelProps to it rather than just all.